msmbuilder/tests/test_ligandfeaturizers.py

```diff
@@ -0,0 +1,104 @@
+import warnings
+
+import mdtraj as md
+import numpy as np
+
+from msmbuilder.featurizer import LigandContactFeaturizer
+from msmbuilder.featurizer import BinaryLigandContactFeaturizer
+from msmbuilder.featurizer import LigandRMSDFeaturizer
+
+
+def _random_trajs():
+    top = md.Topology()
+    c = top.add_chain()
+    r = top.add_residue('HET', c)
+    r2 = top.add_residue('HET', c)
+    r3 = top.add_residue('HET', c)
+    cx = top.add_chain()
+    rx = top.add_residue('HET', cx)
+    for _ in range(10):
+        top.add_atom('CA', md.element.carbon, r)
+        top.add_atom('CA', md.element.carbon, r2)
+        top.add_atom('CA', md.element.carbon, r3)
+    for _ in range(10):
+        top.add_atom('CA', md.element.carbon, rx)
+    traj = md.Trajectory(xyz=np.random.uniform(size=(100, 40, 3)),
+                          topology=top,
+                          time=np.arange(100))
+    ref = md.Trajectory(xyz=np.random.uniform(size=(1, 40, 3)),
+                        topology=top,
+                        time=np.arange(1))
+    return traj, ref
+
+
+def test_chain_guessing():
```

Comment:
could you add a single line after each test explaining what the test is supposed to be testing ?